### PR TITLE
Intro to vast

### PR DESCRIPTION
**Pull request by chrisequalsdev**
Minimal but likely sufficient section to start integrating the _temp_ storage platform into the wiki. Also snuck in patch for #959

### CODE REVIEW

File: _scicomputing/store_scratch.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ title: Scratch File Service
 primary_reviewers: scicomp
 ---
 
+>PLEASE NOTE: the old scratch storage platform is end of life and will be retired soon. Please migrate all scripts and jobs to use [hcp/temp](scicomputing/store_temp.md).
```
Comment (dtenenba) [inline _scicomputing/store_scratch.md:6]: typo: hcp -> hpc

Comment (atombaby) [reply _scicomputing/store_scratch.md]: fixed

File: _scicomputing/store_overview.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Storage Resource | Costs (per TB/month)\*  | Backup Location/Duration | Best Use
...
 Fast | $$$ beyond 5TB per PI |  On campus storage with offsite replication. 7 days of snapshots, daily backups | Large instrument files and data sets that need high performance access to computing resources, Unix file permissions, but not temporary data (such as intermediate files)
 Economy Cloud | $ beyond 100TB per PI |  Offsite storage (cloud) with multi-datacenter replication, 90 day undelete with request to `helpdesk` | Best for archiving large data sets, or primary storage of large files.  Good for PHI or other data that requires encryption and auditing. *Requires Desktop Client to access, see [Object Storage page.](/scicomputing/store_objectstore/)*
-Scratch | No Charge | On campus storage, no snapshots, not backed up | Temporary files, such as those intermediate to a final result that only need to persist during a job, and secondary copies of files used in analysis but with a primary copy on more durable storage  *Appropriate use  can significantly reduce data storage costs- see the [Scratch Storage](/scicomputing/store_scratch/) page.*
```
Comment (atombaby) [inline _scicomputing/store_overview.md:70]: We should keep this line in this table until it's been decomissioned.

File: _scicomputing/store_overview.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ You will need to consider the security needs of your data sets before placing da
 
 This table describes some of the most common security needs for data sets and how or if each storage system implements that security capability.
 
-|  Feature 	                | Secure File	| Fast File | Economy Cloud |  Scratch  	|  OneDrive 	|
```
Comment (atombaby) [inline _scicomputing/store_overview.md:52]: We should keep this line in this table until it's been decomissioned.

**Review by atombaby — CHANGES_REQUESTED** [review]

File: _scicomputing/store_overview.md (hunk 3 above)
Comment (chrisequalsdev) [reply _scicomputing/store_overview.md:52]: Added back scratch

**Review by atombaby — APPROVED** [review]

**Pull request merged**